CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 3.17.2)
 
 project(VULKAN_PROFILES LANGUAGES CXX C)
 
+# Enable GUI folders
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
```

Comment:
FWIW when the cmake min is 3.26 this won't be needed. I added a policy to CMake to make it the default in the future.

https://cmake.org/cmake/help/latest/policy/CMP0143.html